python/lsst/ci/prepare.py

```diff
@@ -264,6 +281,28 @@ def fetch(self, product):
 
         """
 
+        for i in range(self.retries - 1):
+            try:
+                return self._fetch(product)
+            except (GitError, RemoteError, OSError) as e:
+                print('<error>', file=self.out)
+                print(e)
```

Comment:
this is not going to self.out?

Reply:
Ugh, yes, it should be print to `self.out`.